### DISCUSSION TITLE
Sending PDFs from Rocket to Reqwest

### DISCUSSION THREAD

**Discussion by Qualitymix** [rwf2/Rocket#2622] [Questions]
I have hit a roadblock in my current application. 
I cannot seem to successfully transfer the binary data of a PDF from my server side Rocket code to my client side Reqwest code.  Is there a sort of Hello Word example of this type of use case so I can see how it's supposed to be done?

**Comment by SergioBenitez** [comment]
What problem are you having? Your question is far too vague for us to help. Does receiving the file in a browser work? If so, then your issue is likely with reqwest, and I would encourage you to ask for assistance in their support channels.

**Reply by Qualitymix** [reply]
My server side (Rocket) generates my PDF just fine. It's when I attempt to send from rocket to client using Reqwest that the issue occurs. I don't appear to be able to send binary data. I'm wondering what the standard practice is for send binary data of the wire?

**Reply by SergioBenitez** [reply]
Can you read the PDF properly if you make a request from the browser? What about from curl? If yes, then the problem is with reqwest. If no, then please post the relevant logs.
